node/examples/electron/object_detection/README.md

```diff
@@ -0,0 +1,23 @@
+# An Electron.js example of Object_Detection reusing webnn-native
+
+### Install
+
+Firstly, ensure that you have done these steps in [README.md](/node/README.md), then run:
+```bash
+npm install
+```
+
+### Run
+
+```bash
+npm start
+```
+
+You can also run multiple times with "numRuns" to get the median inference time, for example:
+```bash
+npm start numRuns=100
+```
+
+### Screenshot
+
+![screenshot](screenshot.png)
```

Comment:
Do you upload the screenshot?

Reply:
Sorry, I forget to add such a img. I have uploaded it now.